huxley/www/js/stores/DelegateStore.js

```diff
@@ -13,6 +13,7 @@ var {Store} = require('flux/utils');
 
 
 var _schoolsDelegates = {};
+var _committeeDelegates = {};
```

Comment:
This data is getting really complicated. What if we someday want to query delegates by country? Will we have a fourth map? A fifth?

We should figure out a way to query delegates by school and committee, instead of having three different ways of fetching and storing them. School should be pretty easy, since it's a field on the delegate object itself. e.g, `getDelegatesForSchool` or something could be a simple filter statement (if we choose to store them as an array):

```js
return _delegates.filter(d => d.school_id === schoolID);
```

Committees will be a little harder, since we have to go through the assignment store. Any ideas on how we could model this?